### PR TITLE
Continuous Deployment

### PR DESCRIPTION
**Pull request by rathorevaibhav**
## Summary
1. Production and staging builds
2. Removed automerge.yml - we're not using it anymore
3. Cleanups in existing documentation

### CODE REVIEW

File: .github/workflows/production-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      with:
+        host: ${{ secrets.SSH_HOST }}
+        username: ${{ secrets.SSH_USERNAME }}
+        key: ${{ secrets.SSH_PRIVATE_KEY }}
+        envs: SSH_BUILD_DIRECTORY,SSH_POST_BUILD_SCRIPT
+        script: |
+          cd $SSH_BUILD_DIRECTORY
+
+          echo "Enabling maintenance mode"
+          php artisan down
```
Comment (pankaj-ag) [inline .github/workflows/production-deployment.yml:31]: How much downtime we are expecting? Or are we planning to make the late-night build?

File: .github/workflows/staging-deployment.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      with:
+        host: ${{ secrets.SSH_HOST }}
+        username: ${{ secrets.SSH_USERNAME }}
+        key: ${{ secrets.SSH_PRIVATE_KEY }}
+        envs: SSH_BUILD_DIRECTORY,SSH_POST_BUILD_SCRIPT
+        script: |
+          cd $SSH_BUILD_DIRECTORY
+
+          echo "Enabling maintenance mode"
+          php artisan down
```
Comment (pankaj-ag) [inline .github/workflows/staging-deployment.yml:32]: This looks fine.

**Review by pankaj-ag — CHANGES_REQUESTED** [review]

File: .github/workflows/production-deployment.yml (hunk 1 above)
Comment (rathorevaibhav) [reply .github/workflows/production-deployment.yml:31]: Around 5min. This gets trigger on releases. We'll be doing that probably late nights.

Also, these workflows will only trigger after manual approval from the GitHub environment setting.

Comment (pankaj-ag) [reply .github/workflows/production-deployment.yml:31]: Okay, I think it's fine.
We can also use this feature to protect some routes. 
https://stackoverflow.com/questions/30572829/laravel-maintenance-mode-just-for-some-urls

**Review by pankaj-ag — APPROVED** [review]

**Pull request merged**